### ISSUE TITLE
OBS keeps opening in SafeMode

### ISSUE THREAD

**Issue by timetracker2643** [sorayuki/obs-multi-rtmp#381]
OK, I've got the latest version installed, and yet for every other time that I turn on OBS, it keeps wanting to start in safe mode because according to OBS, the program crashed every other time that I turned it off. If I run OBS in SafeMode, the OBS-Multi-rtmp will not appear, meaning that it's been deactivated. However when I run it normally, the OBS-Multi-rtmp is still working, and yet everything is working just fine. I come back to this site every time to see if there's a new version, but so far it's just been the current version. So there is something buggy about this current version that is causing OBS to do a crash when we close it every other time.

**Comment by sorayuki** [comment]
No logs?

**Comment by timetracker2643** [comment]
> No logs?

Would I be getting those from the OBS app?

**Comment by timetracker2643** [comment]
> No logs?

Here's the logs from the OBS app, when I just tried to boot it up, it went into boot in safe mode
https://obsproject.com/logs/F5JdMOOF91wz0aOQ

**Comment by sorayuki** [comment]
require something like it crashes when exit

https://github.com/obsproject/obs-studio/issues/9955

and it seems to be a more general problem

**Comment by msmafra** [comment]
There are some new fixes:
https://github.com/obsproject/obs-studio/releases/tag/30.0.2

**Comment by timetracker2643** [comment]
> There are some new fixes: https://github.com/obsproject/obs-studio/releases/tag/30.0.2

Just wondering, if we've already got the latest version, how do we install the fixes, or are they just there automatically?

**Comment by timetracker2643** [comment]
> > > There are some new fixes: https://github.com/obsproject/obs-studio/releases/tag/30.0.2
> > 
> > 
> > Just wondering, if we've already got the latest version, how do we install the fixes, or are they just there automatically?
> 
> On Windows I don´t know anymore if its automatically. On Linux the Flatpak version updated yesterday. I'm not sure if Snap and Distrobox versions are not updated yet, and package related ones, deb, rpm etc, I imagine, would depend on the distro's update rhythm.

Honestly, after the first install, I've never been told how to update, I just kept replacing the install with a new one, it's not like we're getting told on the Windows side how to update the program with detailed instructions

**Comment by msmafra** [comment]
> > > > There are some new fixes: https://github.com/obsproject/obs-studio/releases/tag/30.0.2
> > > 
> > > 
> > > Just wondering, if we've already got the latest version, how do we install the fixes, or are they just there automatically?
> > 
> > 
> > On Windows I don´t know anymore if its automatically. On Linux the Flatpak version updated yesterday. I'm not sure if Snap and Distrobox versions are not updated yet, and package related ones, deb, rpm etc, I imagine, would depend on the distro's update rhythm.
> 
> Honestly, after the first install, I've never been told how to update, I just kept replacing the install with a new one, it's not like we're getting told on the Windows side how to update the program with detailed instructions

On Windows, Help -> Check for updates https://www.streamscheme.com/how-to-update-obs/. If you installed via Steam, the Steam Client will update it, the Microsoft Store I imagine would do the same.

**Comment by timetracker2643** [comment]
> > > > > There are some new fixes: https://github.com/obsproject/obs-studio/releases/tag/30.0.2
> > > > 
> > > > 
> > > > Just wondering, if we've already got the latest version, how do we install the fixes, or are they just there automatically?
> > > 
> > > 
> > > On Windows I don´t know anymore if its automatically. On Linux the Flatpak version updated yesterday. I'm not sure if Snap and Distrobox versions are not updated yet, and package related ones, deb, rpm etc, I imagine, would depend on the distro's update rhythm.
> > 
> > 
> > Honestly, after the first install, I've never been told how to update, I just kept replacing the install with a new one, it's not like we're getting told on the Windows side how to update the program with detailed instructions
> 
> On Windows, Help -> Check for updates https://www.streamscheme.com/how-to-update-obs/. If you installed via Steam, the Steam Client will update it, the Microsoft Store I imagine would do the same.

Yes, however that doesn't cover the multi-streaming 3rd party app. Updating OBS is all well and good, but updating the OBS-Multi-rtmp is something completely different and won't update with OBS.

**Comment by msmafra** [comment]
Unfortunetly the integrated plugin manager is not a reality yet. It's in discussion since 2018 if not mistaken.
Só, plugins are only updated manually, via some custom install, some via Flatpak or Andi Lipi's manager.

**Comment by mpek** [comment]
> Unfortunetly the integrated plugin manager is not a reality yet. It's in discussion since 2018 if not mistaken. Só, plugins are only updated manually, via some custom install, some via Flatpak or Andi Lipi's manager.

Why would you need that, if an update of a plugin is just an unzip to its location or running an installer?

**Comment by msmafra** [comment]
> > Unfortunetly the integrated plugin manager is not a reality yet. It's in discussion since 2018 if not mistaken. Só, plugins are only updated manually, via some custom install, some via Flatpak or Andi Lipi's manager.
> 
> Why would you need that, if an update of a plugin is just an unzip to its location or running an installer?

I don´t know. It's up to you to know if you need or not.

**Comment by timetracker2643** [comment]
> > Unfortunetly the integrated plugin manager is not a reality yet. It's in discussion since 2018 if not mistaken. Só, plugins are only updated manually, via some custom install, some via Flatpak or Andi Lipi's manager.
> 
> Why would you need that, if an update of a plugin is just an unzip to its location or running an installer?

Because in this case (and with working with Windows 10), when you try to over write what's already there, if you do the normal non-zip side, it asks if you want to over write everything, and then after you click yes, it doesn't do anything. If you do the un-zip thing, then it does the whole asking if you would like to over-write everything that's already there, and when you do that, it just looks like nothing has been done, at least till you restart your OBS. It's a whole big pain in the booty on this.
